hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/event/SoftwareModuleEvent.java

```diff
@@ -41,6 +43,16 @@ public SoftwareModuleEvent(final BaseEntityEventType entityEventType, final Soft
     }
 
     /**
+     * Delete software modules
```

Comment:
I am confused: The other constructor is saying `Creates software module event`. Copy & paste error?